### ISSUE TITLE
Group Readings

### ISSUE THREAD

**Issue by jbeatty88** [Interactive-Phylogeny-Team/InteractivePhylogeny#9]
We might want to start a doc where we can share and discuss readings. What do you guys think? 

We can also start with a list of topics and each person can find readings for a specific topic to put on the doc?

**Comment by maddyras** [comment]
Sarahkate started a bibliography. We could use that or something like it to share the good articles we have found. However, the bibliography doesn't allow us to split the articles up into topics and have discussions.

**Comment by erikhuck** [comment]
@jbeatty88 this sounds potentially like a duplicate of my comment here: https://github.com/Interactive-Phylogeny-Team/InteractivePhylogeny/issues/3#issuecomment-760550842 is issue #3 . But it looks like you add more details. I think this is a good idea. Working off of @sarahkateorr 's bibliography could be good.

**Comment by maddyras** [comment]
https://docs.google.com/document/d/12Z-XmeAPMc-lcQz1akBdOUF4hbPfkgAOfI14StNPyDg/edit?usp=sharing

I decided to make a document so that we could 1) keep track of who found each article and 2) record useful quotes. This way we can easily take turns sharing about the articles we have read in our meetings. (For example, "Maddy, tell us about the three articles you have listed here!") I just copied my citations from Sarahkate's bibliography and added some notes. (Also, we can discuss how many articles we should each find---I just arbitrarily picked six to start.)
